gnomad/utils/vep.py

```diff
@@ -711,6 +711,7 @@ def filter_vep_transcript_csqs(
     genes: Optional[List[str]] = None,
     keep_genes: bool = True,
     match_by_gene_symbol: bool = False,
+    amio_acids: bool = False,
```

Comment:
hmm, this is starting to get a bit over parameterized, and unlike some of the others, I don't think this one will be used that often, what do you think about taking in an optional list of additional filtering criteria, and when it's None, set it to `criteria = [lambda csq: True]`? Then in the pext code pass in 

```        
additional_filtering_criteria = [
            lambda csq: (csq.amino_acids is not None) & (csq.amino_acids != "*")
]
```

Reply:
I'm not sure if I understood clearly, I'll have to commit and push then test the changes..